sysvabi64/sysvabi64.rst

```diff
@@ -720,22 +724,37 @@ open-source compilers.
 
 .. table:: Code Model Support
 
-  +-------+----------------+----------------+
-  | Code  | GCC 11         | Clang 12.0     |
-  | Model |                |                |
-  +=======+================+================+
-  | tiny  | yes            | yes            |
-  |       |                |                |
-  +-------+----------------+----------------+
-  | small | pic and PIC    | pic and PIC    |
-  |       | implemented    | accepted but   |
-  |       |                | PIC used       |
-  +-------+----------------+----------------+
-  | large | no-pic support | no-pic support |
-  |       |                |                |
-  |       |                |                |
-  |       |                |                |
-  +-------+----------------+----------------+
+  +--------+----------------+----------------+
+  | Code   | GCC 11         | Clang 12.0     |
+  | Model  |                |                |
+  +========+================+================+
+  | tiny   | yes            | yes            |
+  |        |                |                |
+  +--------+----------------+----------------+
+  | small  | pic and PIC    | pic and PIC    |
+  |        | implemented    | accepted but   |
+  |        |                | PIC used       |
+  +--------+----------------+----------------+
+  | medium | pic and PIC    | pic and PIC    |
+  |        | proposed       | proposed       |
+  |        |                |                |
+  +--------+----------------+----------------+
+  | large  | no-pic support | no-pic support |
+  |        |                |                |
+  |        |                |                |
+  |        |                |                |
+  +--------+----------------+----------------+
+
+
+Medium code model
+-----------------
+
+Symbols smaller than 64KiB (value TBD) are treated like the small code model.
+Symbols which are larger than 64KiB, are ``common`` or of unknown size use
```

Comment:
I'd prefer Symbols with type ``STT_COMMON`` or of unknown size ... Although I'd expect most people reading the spec to know that, ``common`` might just be confused with its more common (pun intended) English language meaning of frequent.

Reply:
`SHN_COMMON` is more common (pun intended).

In LLVM's integrated assembler, `.common` never uses `STT_COMMON`. binutils defaults to `--elf-stt-common=no` for most configurations and uses `SHN_COMMON` most of the times.